content/_data/changelogs/weekly.yml

```diff
@@ -6293,6 +6293,189 @@
   # pull: 4494 (PR title: Remove references to Azure maven cache)
   # pull: 4502 (PR title: [JENKINS-61102] Use XMLUnit to fix test)
 
+- version: '2.222'
+  date: 2020-02-23
+  changes:
+  - type: major rfe
+    category: major rfe
+    pull: 4463
+    issue: 60920
+    authors:
+    - fqueiruga
+    - daniel-beck
+    message: |-
+      Revamp the layout and icons of the header bar and breadcrumbs.
+      Instances with plugins that depend on details of the Jenkins layout (e.g. Simple Theme Plugin) may experience UI/layout problems.
+      A new experimental header color scheme can be enabled by setting the <code>jenkins.ui.refresh</code> system property to <code>true</code>.
+  - type: major rfe
+    category: major rfe
+    pull: 4463
+    authors:
+    - fqueiruga
+    - daniel-beck
+    message: |-
+      Introduce a new experimental UI that can be enabled by setting the <code>jenkins.ui.refresh</code> system property to <code>true</code>.
+      Currently it includes a new header color scheme, more changes to be added as a part of the UI/UX revamp.
+    references:
+      - pull: 4463
+      - issue: 60920
+      - url: https://github.com/jenkinsci/jep/blob/master/jep/223/README.adoc
+        title: JEP-223
+      - url: https://jenkins.io/sigs/ux/
+        title: Jenkins UX SIG
+  - type: major rfe
+    category: major rfe
+    pull: 4501
+    authors:
+    - daniel-beck
```

Comment:
Since most of the feature was done by @mikecirioli @EstherAF and me before the great PR refactor by @daniel-beck  we would appreciate it if we could be added as co-author of this feature if possible.

Reply:
In fact, @daniel-beck added us as co-authors of the commit with most of the changes https://github.com/jenkinsci/jenkins/commit/55b2bc7f3fe037e46a06deb44b93b02787df28ef

Reply:
Changelog generator did not capture because of the squash-merge (it processes Git metadata ATM, and GitHub does not expose coauthors in REST API). I will add you manually

Reply:
I will also remove @timja. Changelog generator got confused by https://github.com/jenkinsci/jenkins/pull/4501/commits/9ba18f5de0c3b44a4dcd96d3978bc9f3a4c324b1 , we should teach it to ignore the merge commits

Reply:
Created https://github.com/jenkinsci/core-changelog-generator/issues/10 and https://github.com/jenkinsci/core-changelog-generator/issues/9 as follow-ups in the changelog generator